microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -303,8 +303,72 @@ The first number in the group of numbers for an edge in the `edges` array corres
 | Shortcut | An easier-to-read representation of some other path. This type is rarely used. For example, if you use `Function.prototype.bind` to create a bound function with some bound arguments, V8 creates a `JSBoundFunction`, which points to a `FixedArray` (an internal type), which points to each bound argument. When producing a snapshot, V8 adds a shortcut edge from the bound function directly to each bound argument, bypassing the `FixedArray`. |
 | Element | Object properties where the key is a number. |
 
+#### locations
+
+The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of `HeapSnapshot` nodes.
+
+To use this array, the following information is needed:
+
+* `snapshot.meta.location_fields`, to know how many fields each location group has.
+* `snapshot.nodes`, to link a location group back to a node.
```

Comment:
I think  we might almost be able to remove this line. The link to the `nodes` array should be quite obvious with the table you added after.
I think this section would flow better if you removed this, and removed the bulleted list (since there would be only one item in it now).

Reply:
Agreed